### ISSUE TITLE
packaging v2

### ISSUE THREAD

**Issue by Gredin67** [YunoHost-Apps/mautrix_signal_ynh#86]
see https://github.com/YunoHost-Apps/mautrix_whatsapp_ynh/pull/112

**Comment by Gredin67** [comment]
@thardev would you use your experience to do it here ?

**Comment by MayeulC** [comment]
Note that upstream is under heavy rewrite: https://github.com/mautrix/signalgo

**Comment by Gredin67** [comment]
This would change from a python bridge like packaging to a GO like packaging. Maybe @thardev could try to do the packagingV2 at that time or now ?

**Comment by oufmilo** [comment]
Maybe waiting the v0.5.0 is done to rewrite the packaging v2 ?

[https://github.com/mautrix/signal/blob/main/CHANGELOG.md](Changelog)

**Comment by thardev** [comment]
Hi 🙂 

Sorry for not replying here, just forgot about it... I agree with @oufmilo, if the app is going to be rewrote in a significant way, it would be wise to wait until it's done and then do the refactor

**Comment by Gredin67** [comment]
It looks like the main repo has already been moved to Go. I think we can already start to v2 package mautrix_signal by slightly adapting
https://github.com/YunoHost-Apps/mautrix_whatsapp_ynh/pull/112

The main question is : how to handle migration from python to go ? Do we have to remove and reinstall? Will it keep the same DB and handle migration ? Should we use a new _ynh repo and deprecate the old python one?
Just cloning the mautrix_whatsapp_ynh repo would probably be easier.

It would also be a good opportunity to create a main mautrix_go_ynh repo from which all mautrix_go packages would be forked. 

Le 29 janvier 2024 10:38:18 GMT+01:00, Dante ***@***.***> a écrit :

**Comment by oufmilo** [comment]
That would mean using git if you don't want to wait for the release, and that's not a good idea. 

I'm impatient too, but I miss this package in everyday life

**Comment by Gredin67** [comment]
Right, didn't think about it, but I guess the release should come quickly after the repo has been moved.
